packages/core/di/core-di-browser-internal/kibana.jsonc

```diff
@@ -0,0 +1,5 @@
+{
+  "type": "shared-common",
```

Comment:
nit: can we revisit these `type` in our packages and name them according to their scopes:
`shared-browser`/`shared-server`/`shared-common`.